zerocopy-derive/tests/ui/late_compile_pass.rs

```diff
@@ -24,11 +24,13 @@ struct FromBytes1 {
 // AsBytes errors
 //
 
+#[repr(C)]
+struct MyU32(u32);
```

Comment:
Derive `AsBytes` for this? Looks like the errors you're getting are now because this doesn't implement `AsBytes`, not because of padding in `AsBytes1`.

Reply:
That's the intent of this test, I renamed a few things to make it more obvious. Because the check for `AsBytes` happens in a later compile pass, it needs to be tested for in this file.

Reply:
Ah I understand; thanks!